packages/execute-driver/test/e2e/plugin-e2e-specs.js

```diff
@@ -0,0 +1,154 @@
+import path from 'path';
+const APPIUM_HOME = path.resolve(__dirname, '..', '..', '..', 'local_appium_home');
```

Comment:
wait, what's this

Reply:
for this plugin's e2e specs, we want to use an APPIUM_HOME which doesn't overlap with the default APPIUM_HOME, otherwise weird things can happen. we want a sandbox. so this is the sandbox. it's git-ignored in this repo.

the actual path here is essentially `$PLUGIN_ROOT/local_appium_home`